src/ResultsActivity/elements/PayOptions/PayOptions.js

```diff
@@ -47,34 +47,49 @@ function PayOptions({ offenderDetails, updateOffenderAttendance, date, openModal
     )
 
   const allowUpdate = () => {
-    if (selectedOption === 'other' && isWithinLastYear(date)) return true
+    if (selectedOption === 'other' && isWithinLastWeek(date)) return true
     return false
   }
+  const showRadioButton = () => {
```

Comment:
```suggestion
  const showRadioButton = !locked && offenderNo && eventId && isWithinLastWeek(date)
```

should do the same thing

Reply:
I did think about doing that.  Decided to follow the same pattern as allowUpdate() above that had been previously done.  I'll update.